web_scraper.py

```diff
@@ -27,11 +27,12 @@ def __init__(self):
             self.img_url = ""
 
     def __init__(self):
-        self.driver = webdriver.Chrome(ChromeDriverManager().install())
+        
         self.edition = ""
         self.commons = []
 
     def create_commons(self):
+        self.driver = webdriver.Chrome(ChromeDriverManager().install())
```

Comment:
warum hast du die Instanziierung verschoben in die create_commons methode?

Reply:
Der Browser wurde sofort geöffnet, als die Klasse WebScraper() in der main.py einer Variable übergeben wurde. In der init-Methode wurde sie sofort aufgerufen.
Jetzt wird der Browser erst geöffnet, wenn die create_commons() in der main.py aufgerufen wird.
Musste ich so machen, weil ansonsten der Browser immer aufgerufen wird, auch wenn die Datenbank tagesaktuell ist.